### DISCUSSION TITLE
Can neutralinojs.log file be disabled

### DISCUSSION THREAD

**Discussion by Blatman** [neutralinojs/neutralinojs#591] [Q&A] [answered]
Is there maybe an undocumented setting in the neutralino.config.json file that will disable the log file?

**Comment by shalithasuranga** [comment, marked as answer]
Looks like we need to add a new setting to enable/disable the log file. I've added your idea to https://github.com/neutralinojs/roadmap.. Thanks

**Reply by danidre14** [reply]
Please add this setting as soon as possible.

**Comment by Blatman** [comment]
@shalithsuranga - might also be worth having the option to reroute the logger to a folder outside the app bundle (say to a profile folder or such like) which would be better in an end user scenario if logging is to be used.
